show/main.py

```diff
@@ -2,17 +2,17 @@
 
 import click
 import errno
-import getpass
 import json
 import os
 import re
+import sonic_platform
 import subprocess
 import sys
 from click_default_group import DefaultGroup
 from natsort import natsorted
-from tabulate import tabulate
 from swsssdk import ConfigDBConnector
 from swsssdk import SonicV2Connector
+from tabulate import tabulate
```

Comment:
>tabulate [](start = 21, length = 8)

Just curios why it is moved?

Reply:
Alphabetical order by package name.

Reply:
In the latest commit I have reordered import statements into the following groups:

1. Standard library imports
2. Third-party imports
3. SONiC-specific imports
4. Vendor-specific imports